tox_binary_io/src/crypto.rs

```diff
@@ -0,0 +1,52 @@
+use std::convert::TryInto;
+
+use nom::{map, map_opt, named, take};
+
+use crypto_box::{PublicKey, SecretKey, KEY_SIZE};
+
+use super::FromBytes;
+
+
+impl FromBytes for PublicKey {
+    // TODO: use map_from with new nom version
```

Comment:
What's up with new nom version?

Reply:
We have outdated nom version, and this function was added only to the latest version.